<ui version="4.0" >
 <class>UpdateToServerDialog</class>
 <widget class="QDialog" name="UpdateToServerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>453</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>453</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Update To Server</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>160</y>
     <width>431</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Server Hostname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Server Port Number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendServerUpdateButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Update To Server</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverHostNameLineEdit" >
   <property name="geometry" >
    <rect>
     <x>132</x>
     <y>10</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverPortNumberLineEdit" >
   <property name="geometry" >
    <rect>
     <x>132</x>
     <y>40</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>10000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateAllPushButton" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>160</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Get Host Identifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Backup File Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupFileNameLineEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>71</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>/home/backup.txd</string>
   </property>
  </widget>
  <widget class="QToolButton" name="backupSelectFileName" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>70</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="filterUpdatesByDateCheckBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Filter Updates by Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>114</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>2008-04-01</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateToServerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateToServerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
